### ISSUE TITLE
Updating README.md

### ISSUE THREAD

**Issue by pi-squared-4** [Namanl2001/MERN-Gurujii-dev#59]
The current README doesn't fully specify how to open and run the project after cloning it. Some changes are required in README to fully specify it. Let me work on it.

**Comment by guptaji6489** [comment]
I have done some changes have a look.
or something else you want to do?

**Comment by pi-squared-4** [comment]
Yes, like mentioning all the steps like changing to ,env and adding mongourl.

**Comment by guptaji6489** [comment]
Ok, nice.
Please mention all things **step by step** and do for both local as well as online one .

**Comment by guptaji6489** [comment]
Ok, nice.
Please mention all things **step by step** and do for both local as well as online one .

**Comment by pi-squared-4** [comment]
![image](https://user-images.githubusercontent.com/66795444/110946026-6aff0c00-8364-11eb-9d5b-9a55813003f2.png)
![image](https://user-images.githubusercontent.com/66795444/110946043-6e929300-8364-11eb-8203-a9a25954fb37.png)

How does this look?

**Comment by aditisneh** [comment]
Hey @guptaji6489 , while @pi-squared-4 is working on the guidelines part, I wanted to add a cool banner/poster more like a logo for Guruji, to look more attractive  . Kindly assign me if possible.

**Comment by guptaji6489** [comment]
@aditisneh  Please show me like @pi-squared-4  has done

**Comment by guptaji6489** [comment]
@pi-squared-4  how to get MongoURI link is not there.
Please check it

**Comment by aditisneh** [comment]
> @aditisneh Please show me like @pi-squared-4 has done

I didn't get that. What do I show. I can make a banner/logo/gif for guruji. Shall I start..?

**Comment by guptaji6489** [comment]
> > @aditisneh Please show me like @pi-squared-4 has done
> 
> I didn't get that. What do I show. I can make a banner/logo/gif for guruji. Shall I start..?

screenshot or design something

**Comment by pi-squared-4** [comment]
> @pi-squared-4 how to get MongoURI link is not there.
> Please check it

There is a link to a youtube video that explains how to get it.

**Comment by guptaji6489** [comment]
> > @pi-squared-4 how to get MongoURI link is not there.
> > Please check it
> 
> There is a link to a youtube video that explains how to get it.

I didn't  find please have a look.

**Comment by aditisneh** [comment]
> > > @aditisneh Please show me like @pi-squared-4 has done
> > 
> > 
> > I didn't get that. What do I show. I can make a banner/logo/gif for guruji. Shall I start..?
> 
> screenshot or design something

okay shall i share here, otherwise my discord name is Aditi(P)#3051 , can ping me there once :)

**Comment by guptaji6489** [comment]
You can share here

**Comment by aditisneh** [comment]
![logo_guruji](https://user-images.githubusercontent.com/56735634/111200383-b2e19580-85e7-11eb-9ead-d811e2e8f910.png)
This is what I designed for the logo.

**Comment by aditisneh** [comment]
![guruji_gif](https://user-images.githubusercontent.com/56735634/111200949-44e99e00-85e8-11eb-9041-e8cdb0aab677.gif)
This is the gif which will impart the msg of our application

**Comment by aditisneh** [comment]
![Purple Gradient Gamer Girl Animated Logo](https://user-images.githubusercontent.com/56735634/111201068-68ace400-85e8-11eb-8796-4fcaac2f7401.gif)
This  is the gif which will impart the msg of our application

**Comment by aditisneh** [comment]
![Purple Gradient Gamer Girl Animated Logo (1)](https://user-images.githubusercontent.com/56735634/111201537-de18b480-85e8-11eb-8e62-c31d3419130f.gif)
This is the gif which will impart the msg of our application

**Comment by guptaji6489** [comment]
Looking Great
any comments @Namanl2001

**Comment by Namanl2001** [comment]
I would suggest that @aditisneh should open a separate pr to add the banner, to get the gssoc points, it is looking cool but it would be nice if we can have 1/2 more banners to select from. 
For the logo part, we already have a seperate issue, so for now we can add this logo on a temporary basis.
Although the logo design issue is not assigned to anyone yet so @aditisneh if you are interested you can come up with some attractive logo designs. 
Thanks.

**Comment by aditisneh** [comment]
@Namanl2001 @guptaji6489 I am creating a new issue, there i will propose new banners. Do check. Should i make a new issue for logo?

**Comment by guptaji6489** [comment]
@aditisneh I think someone is working on logo

**Comment by guptaji6489** [comment]
@aditisneh  ya you can create new issue

**Comment by aditisneh** [comment]
> @aditisneh I think someone is working on logo

okay

**Comment by mansijain980** [comment]
@Namanl2001 Can i work on this issue?

**Comment by Namanl2001** [comment]
> 
> 
> @Namanl2001 Can i work on this issue?

What kind of changes you want to add?

**Comment by mansijain980** [comment]
> ![image](https://user-images.githubusercontent.com/66795444/110946026-6aff0c00-8364-11eb-9d5b-9a55813003f2.png)
> ![image](https://user-images.githubusercontent.com/66795444/110946043-6e929300-8364-11eb-8203-a9a25954fb37.png)
> 
> How does this look?

@Namanl2001 sir actually i saw nd this is perfectly fine ... jst wanna add these with some more changes .like testing on mobile 
 we can give commands so that they can easily run on mobile as well .

**Comment by AmanPachori** [comment]
@Namanl2001 what things to be updated in readme file can you please specify

**Comment by Namanl2001** [comment]
That's your part to do @AmanPachori :)
Go through the previous comments, current readme. Try to setup the project and note down the difficulties you faced in the readme.
